tests/test-cases/Group1-Docker-Commands/1-08-Docker-Logs.robot

```diff
@@ -69,7 +69,9 @@ Docker logs backward compatibility
     Should Contain  ${output}  container ${id1} does not support '--timestamps'
```

Comment:
Commenting here since I can't seem to add a comment above. Why is the `Docker logs backward compatibility` test not using harbor?